tests/check_odov2.sh

```diff
@@ -206,7 +206,7 @@ for stack in $STACKS; do
 
   # Skipping the java-wildfly-bootable-jar stack right now since it's broken.
   # TODO: Uncomment once fixed.
-  if [ $stack != "java-wildfly-bootable-jar" ]; then
+  if [[ $stack != "java-wildfly-bootable-jar" && $stack != "java-quarkus/1.3.0" && $stack != "java-quarkus/1.4.0" ]]; then
```

Comment:
`java-quarkus v1.4.0` is not tested by odov2 with or without this statement because it uses a 2.2.x devfile schema, though I'm also good leaving it in anyway since it appears the entire stack is failing the checks right now.

Reply:
Ah yes that is true. I just added that there so it was consistently being skipped in both `odov2` and `odov3`. I feel that having it setup like that makes it easy to ensure nothing weird happens in the tests, but if you want to remove it I can do that

Reply:
> I feel that having it setup like that makes it easy to ensure nothing weird happens in the tests

:+1: